src/main/java/ledger/user_interface/ui_controllers/TransactionPopupController.java

```diff
@@ -130,22 +131,32 @@ private void printStackTrace(Exception e) {
      * @return a new Transaction object consisting of user input
      */
     public Transaction getTransactionSubmission() {
-        try {
+
             LocalDate localDate = this.datePicker.getValue();
             Instant instant = Instant.from(localDate.atStartOfDay(ZoneId.systemDefault()));
             this.date = Date.from(instant);
 
             this.pending = this.pendingCheckBox.isSelected();
 
             this.payee = fromBoxPayee(this.payeeText.getValue());
+            if (InputSanitization.isInvalidPayee(this.payee)){
+                this.setupErrorPopup("Invalid Payee entry.", new Exception());
```

Comment:
Shouldn't matter unless you want the exceptionMessage in the error box. I do this in a couple places because we have to provide the popup an exception, even if there wasn't one.